mkdocs.yml

```diff
@@ -253,6 +253,8 @@ plugins:
         "models/extras/gitrepository.md": "user-guide/platform-functionality/gitrepository.md"
         "models/extras/jobhook.md": "user-guide/platform-functionality/jobs/jobhook.md"
         "models/extras/joblogentry.md": "user-guide/platform-functionality/jobs/models.md"
+        "models/extras/jobqueue.md": "user-guide/platform-functionality/jobs/jobqueue.md"
+        "models/extras/jobqueueassignment.md": "user-guide/platform-functionality/jobs/jobqueueassignment.md"
```

Comment:
Technically these should be located later on, around line 311-313, since they're new in post-2.x and for whatever reason we decided to divide up the redirects in this file accordingly.